tests/testthat/test-build-articles.R

```diff
@@ -61,13 +62,15 @@ test_that("articles don't include header-attrs.js script", {
 
 test_that("can build article that uses html_vignette", {
   pkg <- local_pkgdown_site(test_path("assets/articles"))
+  suppressMessages(init_site(pkg))
 
   # theme is not set since html_vignette doesn't support it
   expect_snapshot(expect_error(build_article("html-vignette", pkg), NA))
 })
 
 test_that("can override html_document() options", {
   pkg <- local_pkgdown_site(test_path("assets/articles"))
+  suppressMessages(init_site(pkg))
```

Comment:
would it be too cumbersome to have an `init` argument to `local_pkgdown_site()`?

Reply:
Yeah, I'll think about that as I continue to refactor the tests.